docs/server/commands.rst

```diff
@@ -346,6 +346,129 @@ By default, :ref:`commands#test_checks` tests all existing checks. When
 against.
 
 
+.. _commands#vfolders:
+
+Virtual Folders
+---------------
+
+These commands allow you to perform tasks with virtual folders from the command
+line.
+
+
+.. _commands#add-virtual-folders:
+
+add_vfolders
+^^^^^^^^^^^^
+
+.. versionadded:: 2.7.0
+
+This command allows you to create **virtual folders** from a JSON file. If the
+specified virtual folders already exist and the data differs, then they are
+updated with the provided data.
+
+The JSON file consists in a list of virtual folder definitions with the
+following fields:
```

Comment:
Can we split the expected format docs away from the command reference and put it in its own section? Since there's already a *docs/features/virtual_folders.rst* file and nothing links to the format spec, it's rather surprising to find this information along the commands.